index.html

```diff
@@ -152,140 +152,268 @@ <h2 class="title">Tonic</h2>
-                  </label>
-                </summary>
-                <div class="summary-down">
-                  <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-up"><polyline points="18 15 12 9 6 15"></polyline></svg>
-                  </div>
-              </details>
-            </div>
+                <label class="summary-up">
+                  <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    width="24"
+                    height="24"
+                    viewBox="0 0 24 24"
+                    fill="none"
+                    stroke="currentColor"
+                    stroke-width="2"
+                    stroke-linecap="round"
+                    stroke-linejoin="round"
+                    class="feather feather-chevron-down"
+                  >
+                    <polyline points="6 9 12 15 18 9"></polyline>
+                  </svg>
+                </label>
+              </summary>
+              <div class="summary-down">
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  width="24"
+                  height="24"
+                  viewBox="0 0 24 24"
+                  fill="none"
+                  stroke="currentColor"
+                  stroke-width="2"
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                  class="feather feather-chevron-up"
+                >
+                  <polyline points="18 15 12 9 6 15"></polyline>
+                </svg>
+              </div>
+            </details>
           </div>
-             <!--********************************************************-->
+        </div>
+        <!--********************************************************-->
+      </div>
+    </section>
+    <!--About my self section end-->
 
+    <!--Contact me section -->
+    <div class="contact-background"></div>
+    <div class="mainContainer contact-form">
+      <section class="secondContainer">
+        <h1>Contact me</h1>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </section>
+      <div class="form-container">
+        <form
+          action="https://formspree.io/f/mgedpajj"
+          method="post"
+          class="form"
+        >
+          <div class="form-group">
+            <input
+              type="text"
+              name="name"
+              id="name"
+              class="form-controler"
+              maxlength="30"
+              placeholder="Please enter your name"
+              required
+            />
           </div>
-        </section>
-      <!--About my self section end-->
-
-
-
-
-
```

Comment:
***[OPTIONAL]***

- Please, let's keep our codebase clean by removing the unnecessary added spaces to control the use of indentation and to make our codebase more readable

Reply:
Thanks for the comment, but I do not think I had this in my code. if there was, so why I did not receive any linter error?